Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:2.7
+
+MAINTAINER rail@mozilla.com
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get -q update && \
+    apt-get -q --yes install sqlite3 && \
+    apt-get clean
+
+WORKDIR /app
+
+COPY . /app
+WORKDIR /app
+RUN pip install -r requirements/compiled.txt -r requirements/dev.txt -r requirements/prod.txt
```

Comment:
I don't think you want the dev requirements in the Docker image; they're only used for running tests.